src/routes/docs/tooling/command-line/installation/+page.markdoc

```diff
@@ -84,12 +100,15 @@ appwrite client --selfSigned true
 ```
 {% /info %}
 
+{% info title="Correct CLI Version Required" %}
+All commands only work with **version 2.0** of the CLI and will **not** work on previous versions. Update [here](https://www.npmjs.com/package/appwrite-cli-beta).
```

Comment:
This can be placed and phrased better. In the installation page we can safely assume people have the latest version. This note will be more beneficial in the top of the commands page. A better way to show this would be to focus on encouraging devs to run on the latest version instead of saying this only works with a specific release.

Reply:
Updated to:
```
{% info title="CLI Version" %}
All commands are compatible with the latest version of the CLI. [Update your CLI](/docs/tooling/command-line/installation#update-your-cli) to the latest version.
{% /info %}
```